rules/DeadCode/Rector/Expression/RemoveDeadStmtRector.php

```diff
@@ -58,6 +58,10 @@ public function refactor(Node $node)
             return $this->removeNodeAndKeepComments($node);
         }
 
+        if ($livingCode === [$node->expr]) {
```

Comment:
That´s it? I don´t get it.

Reply:
It means that the code that should be kept alive (not removed) is the same as current node.
Nothing to be removed, just return `null`

Reply:
This rule would deserve more refactoring to improve readability and logic behind it, but we have more important things to do :+1: